src/SFA.DAS.ApplyService.Domain/Ukrlp/ProviderDetails.cs

```diff
@@ -32,6 +32,11 @@ public bool VerifiedbyCharityCommission
             }
         }
 
+        public ProviderContact PrimaryContactDetails
+        {
+            get { return ContactDetails.FirstOrDefault(x => x.ContactType == "L"); }
```

Comment:
minor, minor point.  Maybe define "L" as a named variable describing what it is? It's minor as it is used in one place, where it's use is evident, so I'll leave that with you.